woocommerce/woocommerce
# Add experimental product task feature flag & experimental-products component

### All Submissions:






### Changes proposed in this Pull Request:



Closes #32772.

Added `experimental-products-task` feature flag and import `experimental-products` component when the flag is on.

### How to test the changes in this Pull Request:

1. Run `pnpm nx dev woocommerce-admin` or `pnpm nx start woocommerce-admin`
2. Go to **WooCommerce > Home**  & click the "Add my products" task
3. Observe that the existing product task is displayed
5. Open the `plugins/woocommerce/client/admin/config/development.json` and change `experimental-products-task` to `true`
6. Repeat 1~2
7. Observe that `Experimental products` text is shown on the page

### Other information:




### FOR PR REVIEWER ONLY:
